src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-Components.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Developer Introduction to the Infusion Framework - Components
+layout: default
+category: Tutorials
+---
+
+Programs using Infusion structure their data, functionality and interaction as various components defined using a common syntax.
+
+Components are [JavaScript objects that have certain characteristics](../UnderstandingInfusionComponents.md). Components can be freely combined together to form new components using the [grades system](../ComponentGrades.md) ("grade" is the term used in Infusion for a sort of blueprint, in this case one that holds a component's default configuration), or organized into relationships with one another via [subcomponents](../SubcomponentDeclaration.md).
+
+ There are three base component grades to be aware of that any new components you create with Infusion will likely have at their root:
+
+ * [`fluid.component`](../ComponentConfigurationOptions.md#options-supported-by-all-components-grades), the simplest type with support for events and public function definition (called [invokers](../Invokers.md) in Infusion).
+ * [`fluid.modelComponent`](../ComponentConfigurationOptions.md#model-components), which adds support for mutable models, model events, and model state coordination and relay between different components.
+ * [`fluid.viewComponent`](../ComponentConfigurationOptions.md#view-components), which supplement model components with functionality to bind to the DOM of a web page.
+
+Infusion's approach to component-based development emphasizes declarative configuration, loose coupling and flexible hierarchies. The framework manages component lifecycle and scoping and provides many supports to allow components to work in collaboration with one another.
+
+A new Infusion component grade is defined using the `fluid.defaults` function and a very basic "Hello, World!" component might look like this:
+
+<div class="infusion-docs-note">You can check out the [Live Example of the code below on CodePen](http://codepen.io/waharnum/pen/oBYvwx?editors=1111)</div>
```

Comment:
The note is missing the `<strong>Note:</strong>` at the beginning. Also the link should be written using HTML because it is in a `<div>`